devel/tests/test_user.py

```diff
@@ -1,8 +1,8 @@
 from django.contrib.auth.models import User
 from django.test import TestCase
 
-from .utils import UserFinder
-from .models import UserProfile
+from devel.utils import UserFinder
+from devel.models import UserProfile
```

Comment:
Without it, the PR will obviously not work and we want the tests moved in subdiir/tests/test_component.py